guides/concepts/transactions.md

```diff
@@ -49,13 +49,27 @@ Between ledger closings, all the nodes in the network are collecting transaction
-If the preliminary checks pass, then stellar-core propagates the transaction to all the other servers to which it's connected. In this way, a valid transaction is flooded to the whole Stellar network.
+4. **Propagating**: Once stellar-core receives a transaction, either from a user or another
+   stellar-core, it does preliminary checks to see if the transaction is valid by [checking the
+   validity of its operations](./operations.md#validity-of-an-operation). If the preliminary checks
+   pass, then stellar-core propagates the transaction to all of the other servers to which it's
+   connected. In this way, a valid transaction is flooded to the whole Stellar network. If a
+   transaction fails any of these checks, the sequence number of the account remains unchanged.
 
-5. **Including in a transaction set**: When it's time to close the ledger, stellar-core takes all the transactions it has heard about since last ledger close and collects them into a transaction set. If it hears about any incoming transactions now, it puts them aside for next ledger close.
-Stellar-core nominates the transaction set it has collected. SCP resolves the differences between the various transaction sets proposed and decides on the one transaction set that the network will apply.
+5. **Including in a transaction set**: When it's time to close the ledger, stellar-core takes all
+   the transactions it has heard about since the last ledger close and collects them into a
+   transaction set. If it hears about any incoming transactions now, it puts them aside for the
+   next ledger close. Stellar-core nominates the transaction set it has collected. Finally, SCP
+   resolves the differences between the various transaction sets proposed and decides on the one
+   transaction set that the network will apply.
 
-6. **Application**: Once SCP agrees on a particular transaction set, that set is applied to the ledger. At this point, a fee is taken from the source account for every transaction in that set. Operations are attempted in the order they occur in the transaction. If any operation fails, the whole transaction fails, and the effects of previous operations in that transaction are rolled back. After all the transactions in the set are applied, a new ledger is created and the process starts over.
+6. **Application**: Once SCP agrees on a particular transaction set, that set is applied to the
+   ledger. At this point, a fee is taken from the source account for every transaction in that set,
+   and the source account's sequence number is increased by 1 for each of their transactions in
```

Comment:
this is wrong: sequence numbers are updated when transactions are applied.
Some reconciliation of https://github.com/stellar/docs/blob/master/guides/concepts/transactions.md#sequence-number and this section is needed